### PR TITLE
Create run-full-model-execution-tests-nightly.yml w/ 13 tests passing

### PR DESCRIPTION
**Pull request by kmabeeTT**
### Ticket
None

### Problem description
 - We have more passing full-model-execution coming online, and desire to keep them out of push/pr and only in nightly, so need to create a home for them.  Also, since we have a bunch of new ones passing now, would like to keep them protected. I suspect in the future this will be more regular occurrance and could benefit from automated flow to do this that is WIP.

### What's changed
 - Create new file run-full-model-execution-tests-nightly.yml used only by nightly-tests.yml that contains 13 tests passing
 - They are part of set of 23 tests passing op-by-op flow to execute but 10 other tests have runtime issues or pcc fails, so excluding
 - Ideally wanted these to be inside existing yaml and have matrix be dynamically configured for push vs nightly, but not simple.
 - 2 parallel jobs take about 10 minutes each

### Checklist
- [x] New/Existing tests provide coverage for changes

Link to passing run at TOTT before creating this branch and PR: https://github.com/tenstorrent/tt-torch/actions/runs/14024930263/job/39262196611 (I moved 2 tests from group 2 to group 1 after it ran to better balance load).

### CODE REVIEW

File: .github/workflows/run-full-model-execution-tests-nightly.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+jobs:
+  tests:
+    timeout-minutes: 120
+    strategy:
+      fail-fast: false
+      matrix:
+        build: [
+          {
+            runs-on: wormhole_b0, name: "eval_1", tests: "
+                  tests/models/torchvision/test_torchvision_image_classification.py::test_torchvision_image_classification[full-eval-vgg19]
```
Comment (AleksKnezevic) [inline .github/workflows/run-full-model-execution-tests-nightly.yml:29]: Are any of these duplicated in the compile tests?

Comment (kmabeeTT) [reply .github/workflows/run-full-model-execution-tests-nightly.yml:29]: Yep a few. When that PR gets merged tonight will update this one to remove the duplicates from the e2e compile test list yaml.

**Comment by jameszianxuTT** [comment]
I think that it is a bit risky to duplicate the full-model-execution tests just to support different subsets of tests. I'm imagining someone may accidentally modify one of these files and not make a corresponding modification to the other, which could lead to some errant behaviour. 

As you say:
> Ideally wanted these to be inside existing yaml and have matrix be dynamically configured for push vs nightly, but not simple.

I'll get on this soon, it would be a relevant feature to #443. There should be precedent in other repos' CI for this dynamic matrix configuration. For now, I think we could go with this stopgap solution until that dynamic test matrix generation infra can be set up.

**Review by jameszianxuTT — APPROVED** [review]

File: .github/workflows/run-full-model-execution-tests-nightly.yml (hunk 1 above)
Comment (kmabeeTT) [reply .github/workflows/run-full-model-execution-tests-nightly.yml:29]: Done. Merged the PR to add bunch of tests to compile list, and rebased this PR to remove duplicates fromt the compile list (which was all 13 tests actually)... so this PR essentially promotes them from nightly compile list to new nightly execute list.  Resolving.

**Comment by kmabeeTT** [comment]
thx @jameszianxuTT for review!

> I think that it is a bit risky to duplicate the full-model-execution tests just to support different subsets of tests. I'm imagining someone may accidentally modify one of these files and not make a corresponding modification to the other, which could lead to some errant behaviour.

Yeah, though to be clear it's not the tests themselves that are duplicated, just the body of this file and it's actions etc. Agreed though, don't love it.  Had a few different ideas to solve it, but ended up getting ugly/complicated and not as quick as this... let's discuss this week :)
